campaignion_newsletters/campaignion_newsletters.variable.inc

```diff
@@ -23,7 +23,7 @@ function campaignion_newsletters_variable_info($options) {
     'default' => 5,
     'localize' => FALSE,
   ];
-  $v['campaignion_newsletters_unsubscribe_unknown'] = [
```

Comment:
I think the naming somewhat consistent:

- `campaignion_opt_in` talks about “opt-ins” and “opt-outs”.
- `campaignion_newsletters` talks about subscriptions, subscribe and unsubscribe.

Why do you think it is inconsistent?

Reply:
Because `campaignion_newsletters_unsubscribe_unknown` is not used by any code, instead `variable_get()` [here](https://github.com/moreonion/campaignion/blob/7.x-2.x/campaignion_newsletters/src/Component.php#L41) reads `campaignion_newsletters_opt_out_unknown`.

Reply:
Ok, then IMHO the variable read is at fault.

Reply:
Changed the pull request. But now we’ll need a migration for already existing variables…

Reply:
True. So an update hook is needed, right?